src/metabase/types.clj

```diff
@@ -0,0 +1,94 @@
+(ns metabase.types)
```

Comment:
Is the main rationale for "type/Blah" vs just "Blah" to let them map to clojure-land types directly?